distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -235,6 +235,8 @@ type JS_Object
                             Error.throw (Illegal_Argument.Error "Unable to build a "+target_type.to_text+" - the constructor "+constructor.name+"'s fields "+missing_fields.to_display_text+" not found in the source object.")
                     _ -> Error.throw (Illegal_Argument.Error "Unable to build a "+target_type.to_text+" cannot find unique constructor.")
 
+    set_value : Text -> Any -> JS_Object
+    set_value self:JS_Object key:Text value = JS_Object.Value (set_value self.js_object key value.to_js_object)
```

Comment:
This needs a comment, some documentation explaining what it does, or at least a `PRIVATE` tag if  its internal.

Reply:
Done